bin/cgi/viewvc.cgi

```diff
@@ -48,6 +48,17 @@ else:
 
 #########################################################################
 #
+# Uncomment to disable robots. Helps to avoid useless load.
+#
+#
+#PATH_INFO = os.environ.get('PATH_INFO', '')
+#if PATH_INFO == '/robots.txt':
+#    print "Content-Type: text/plain\n\nUser-agent: *\nDisallow: /\n"
+#    sys.exit(0)
+
```

Comment:
I can see the value of this, but we try to avoid asking folks to comment/uncomment code in ViewVC.  (Yeah, I realize that there's some precedent here with the nice() call that follows.)

Is this feature something that would be more generally useful to all the ViewVC deployment scenaries (CGI, WSGI, FCGI, mod_python, etc.)?  Seems like something that could be implemented in `lib/viewvc.py` with an option in `viewvc.conf` to toggle on/off.